service_container/import.rst

```diff
@@ -80,7 +80,8 @@ a relative or absolute path to the imported file:
         # config/services.yaml
         imports:
             - { resource: services/mailer.yaml }
-
+            # If you want to import a directory, use the next configuration
+            - { resource: services/ }
```

Comment:
Tbh, it does look kind of weird with the unquoted trailing `/`, but guess this is how they implemented this functionality.